README.md

```diff
@@ -2,7 +2,61 @@
 [![codecov](https://codecov.io/gh/allenai/allennlp/branch/master/graph/badge.svg)](https://codecov.io/gh/allenai/allennlp)
 # AllenNLP
 
-A [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) natural language processing toolkit using state-of-the-art deep learning models.
+An [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) NLP research library, built on PyTorch,
+for developing state-of-the-art deep learning models.
+
+## Quickstart
+
+The fastest way to get an environment to run AllenNLP is with Docker.  Once you have [installed Docker](https://docs.docker.com/engine/installation/)
+just run `docker run -it --rm allennlp/allennlp-cpu` to get an environment that will run on the cpu (use `allennlp-gpu` if you have a CUDA-supported GPU).
+Now you can do any of the following:
+
+* Run a model on example sentences with `allennlp/run bulk`.
+* Start a web service to host our models with `allennlp/run serve`.
+* Interactively code against AllenNLP from the Python interpreter with `python`.
+
+## What is AllenNLP?
+
+Built on PyTorch, AllenNLP makes it easy to design and evaluate new deep
+learning models for nearly any NLP problem, along with the infrastructure to
```

Comment:
AllenNLP Features:

- **Hyper-modular and Lightweight**. Use parts of it which you like seamlessly with Pytorch.
- **Extensively tested and easy to extend**. We welcome contributions and new models.
- **Takes padding and masking seriously**, making it easy to implement technical models correctly without pain.
- **Experiment Friendly**. Run reproducible experiments with comprehensive logging from JSON.